### DISCUSSION TITLE
Create the copyparty-sfx.py file

### DISCUSSION THREAD

**Discussion by AlexandreMuller** [9001/copyparty#127] [Q&A] [answered]
How can I make my modifications and create the copyparty-sfx.py file?

**Comment by 9001** [comment, marked as answer]
Hey! Thanks for the interest, curious to see what you're up to :>

I tried to document the process here: https://github.com/9001/copyparty/blob/hovudstraum/docs/devnotes.md#building

Assuming you're not planning to modify any of the third-party dependencies, then the following steps work on fedora:

```bash
dnf install git python3
git clone https://github.com/9001/copyparty/
cd copyparty/
./scripts/make-sfx.sh dl-wd fast 
python3 dist/copyparty-sfx.py --version
```

if you do need to modify any of the third-party dependencies, then I just realized that the devnotes doesn't mention how to do this. You can build the deps from source with `make -C scripts/deps-docker` and then afterwards `./scripts/make-sfx.sh fast` (without the `dl-wd`) to use your local deps instead of borrowing from the last release.

Good luck :>

**Reply by AlexandreMuller** [reply]
It worked perfectly, thank you!
